src/Microsoft.Data.SqlClient/tests/ManualTests/Microsoft.Data.SqlClient.ManualTesting.Tests.csproj

```diff
@@ -72,6 +55,21 @@
     <Compile Include="AlwaysEncrypted\TestFixtures\Setup\DummyKeyStoreProvider.cs" />
     <Compile Include="AlwaysEncrypted\TestFixtures\Setup\DummyProviderMasterKey.cs" />
     <Compile Include="AlwaysEncrypted\TestFixtures\Setup\CertificateUtility.cs" />
+    <Content Include="AlwaysEncrypted\TestFixtures\Setup\TCECryptoNativeBaseline.txt">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <Link>TCECryptoNativeBaseline.txt</Link>
+    </Content>
+    <Content Include="AlwaysEncrypted\TestFixtures\Setup\TCECryptoNativeBaselineRsa.txt">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <Link>TCECryptoNativeBaselineRsa.txt</Link>
+    </Content>
+    <Compile Include="AlwaysEncrypted\TestTrustedMasterKeyPaths.cs" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetsWindows)' == 'true' AND ('$(TestSet)' == '' OR '$(TestSet)' == 'AE')">
```

Comment:
I agree with the way that the csproj has been restructured, but I wonder if there's room to go further in a future PR. If we switched away from conditional compilation and started using xUnit traits to filter the unit tests we run, the functional and manual test binaries would always be identical. We could have a very small set of compilation stages in the pipeline (one for each platform) and then pass the results of these stages to the broader set of test stages.

If that works, we could cut each test stage by 4-5 minutes, and if for some reason the tests don't even compile, we might never need to run the test stages at all.